en/streaming-integrator/docs/examples/handling-requests-with-errors.md

```diff
@@ -152,9 +181,11 @@ To simulate REST API calls for the purpose, follow the procedure below:
 
     ![Imported Collections](../../images/handling-requests-with-errors/Postman.png)
     
-3. Under **Siddhi-Re-Stream Events**, select **Invalid Attribute** and click **Send**. This executes the collection to send an event with erroneous mapping. The error is indicated as follows:
+3. Under **Siddhi-Re-Stream Events**, select **Invalid Attribute** and click **Send**. 
+
+    This executes the collection to send an event. To view the event, go to the **Body** tab  where the event is displayed as shown below.
```

Comment:
Shall we remove things related to Postman and replace with the CURL commands? (Only the commands to send events to the endpoints mentioned in Siddhi apps is enough)